packages/metal-dom/src/domData.js

```diff
@@ -35,6 +27,27 @@ class domData {
 	static has(element) {
 		return !!element[METAL_DATA];
 	}
+
+	/**
+	 * Sets Metal.js's data for the given element.
+	 * @param {!Element} element
+	 * @param {string=} name Optional Property from the data to be returned.
+	 * @param {*=} value Optional value to the set the requested property
+	 *     to if it doesn't exist yet in the data.
+	 * @return {!Object}
+	 */
+	static set(element, name, value) {
+		if (!element[METAL_DATA]) {
+			element[METAL_DATA] = {};
+		}
+		if (!name || !value) {
```

Comment:
Wouldn't this check prevent someone from passing a falsy value?

```
domData.set(element, 'foo', true)
// foo === true
domData.set(element, 'foo', false)
// foo === true
```

Reply:
I'll correct this!